src/app/pages/waiting-room-to-car/cat-b/waiting-room-to-car.cat-b.page.html

```diff
@@ -73,9 +73,6 @@
       </ion-grid>
     </form>
   </div>
-</ion-content>
-
-<ion-footer [hidden]="pageState.eyesightTestFailed$ | async">
```

Comment:
are we not losing functionality here.  looks like this needs to be hidden if eyesight test failed which your removing